jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -8,6 +8,9 @@
 {{ $XMPP_PORT := .Env.XMPP_PORT | default "5222" -}}
 {{ $XMPP_SERVER := .Env.XMPP_SERVER | default "xmpp.meet.jitsi" -}}
 {{ $XMPP_SERVERS := splitList "," $XMPP_SERVER -}}
+{{ $ENABLE_VOSK_TRANSCRIPTION := .Env.ENABLE_VOSK_TRANSCRIPTION | default "0" | toBool  -}}
+{{ $ENABLE_GCLOUD_TRANSCRIPTION := .Env.ENABLE_GCLOUD_TRANSCRIPTION | default "0" | toBool  -}}
+{{ $VOSK_SERVER := .Env.VOSK_SERVER}}
```

Comment:
I think this needs a default

Reply:
> 

Maybe my jitsi setup deviated too far, since I previously scripted together the VOSK support by hand. I will try a fresh and clean install with the commit and see, what doesn't work. I may have forgotten a var or default, like rouaidakacem mentioned

Reply:
Also I do not see the use of the varaible ENABLE_GCLOUD_TRANSCRIPTION it is defined in sip-communicator.properties but it is not used.